tests/contrib/flask/test_hooks.py

```diff
@@ -129,7 +129,7 @@ def before_first_request():
 
         # Assert correct parent span
         if flask_version >= (2, 2, 0):
-            self.assertEqual(parent.name, "flask.request")
+            self.assertEqual(parent.name, "flask.application")
```

Comment:
is this a breaking change? the root span is changing from `flask.request` to `flask.application`?

Reply:
or no, this is a new additional span being added for Flask integration?

Reply:
This shouldn’t be a breaking change. The flask.request will still have the same tags and still serve as the top level span. This PR just adds 2 more spans to the trace (application and response spans).

Reply:
ok, thanks, that is what I thought